src/api/cvc4cpp.h

```diff
@@ -1151,6 +1151,76 @@ class CVC4_PUBLIC Term
   // to the new API. !!!
   const CVC4::Node& getNode(void) const;
 
+  // mkTuple
+  bool isTuple() const;
```

Comment:
I'm not sure I'd insist on `is` functions for each `mk`.

Concretely i dont think we need isTuple, isTrue, isFalse, isPi, isValueReal, isRegexpEmpty, isRegexpSigma, isEmptySet, isEmptyBag, isSepNil, isChar, isEmptySequence, isUniverseSet, isConstArray, isPosInf, or anything below it.

Reply:
Conversely, I think we should consider:

std::unordered_set<api::Term> isSet / getSet, std::vector<api::Term> isSequence / getSequence

Reply:
I added `isX` and `getX` for sets, sequences and tuples.
I would think that we want to have such functions for everything that we can construct via the API... we should discuss this internally.

Reply:
From a user perspective, I think that we do. How else would you query what kind of value it is?